users/views.py

```diff
@@ -8,11 +8,23 @@
 from rest_framework.authtoken.models import Token
 from rest_framework.response import Response
 from users.models import UserProfile
+from django.core.mail import send_mail
+from HipoProject.settings import EMAIL_HOST_USER
+from django.template.loader import render_to_string
 
 
 class UserRegistrationView(generics.CreateAPIView):
     serializer_class = CreateUserSerializer
 
+    def post(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        email_from = EMAIL_HOST_USER
+        send_mail(subject='Test', message='Test message', from_email=email_from,
```

Comment:
Do you use `message` in template ?
You can add an utility/wrapper method for sending email and do not sent `from_email`, `fail_silently` etc. parameters each time.